alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -100,8 +104,12 @@ export class BoardComponent implements OnInit {
         data: application,
       });
 
-      dialogRef.afterClosed().subscribe(() => {
+      dialogRef.afterClosed().subscribe((isDirty) => {
         this.setUrl();
+
+        if (isDirty && this.selectedBoardCode) {
+          this.loadApplications(this.selectedBoardCode);
+        }
       });
     } catch (err) {
       console.log(err);
```

Comment:
probably we need to create a ticket to replace console.log with appropriate toast messages where applicable

Reply:
added a toast message for this catch block